site_scons/prereq_tools/base.py

```diff
@@ -914,9 +914,9 @@ def add_options(self):
         AddOption('--build-deps',
                   dest='build_deps',
                   type='choice',
-                  choices=['yes', 'no', 'build-only'],
+                  choices=['yes', 'no', 'only'],
```

Comment:
What did build-only do?  Was that for fire-walled sites where we need to build but cannot download?

Reply:
You are correct.  I just reverted this change.   There could potentially be a matrix here but for now, just leave it as only and build-only (no build-only-only, lol)